report/2.Analysis/SensorsAndMotorsNXT.tex

```diff
@@ -9,33 +10,38 @@ \subsection{Ultrasonic Sensor}
 The sensor is capable of both measuring ambient light, such as the light levels of a room, and reflected light by using the built-in LED.
 
-\subsection{Dual range, triple zone infrared obstacle detector}
+\subsection*{Dual range, triple zone infrared obstacle detector}
 The dual range, triple zone infrared obstacle detector, also known as NXTSumoEyes, is an obstacle sensor with an effective range of up to 20 centimeters using infrared beams.
 The major usability difference between the NXTSumoEyes and the ultrasonic sensor is that the NXTSumoEyes works with three individual detection zones of 20 degrees each, allowing for a more detailed response on the location of the detected object.
 
-\subsection{IRRD-T sensor}
+\subsection*{IRRD-T sensor}
 The infrared relative distance sensor, IRRD, measures the distance to an object with an effective range of up to 60 centimeters by measuring reflected infrared light, in a similar faction to the ultrasonic sensor.
 
-\subsection{HPMR Infrared Distance Sensor}
+\subsection*{HPMR Infrared Distance Sensor}
 The high precision medium range infrared distance sensor work in the same way as the IRRD-T sensor previously described.\\
 However, this sensor allows detection of objects with a range of 10 to 80 centimeters, by using a Sharp GP20A02YK sensor, which is slightly more powerful than the sensor used in the IRRD-T sensor.
 
-\subsection{Motors}
+\subsection*{Motors}
 The NXT series offers a variety of motors in two categories: DC motors and servo motors.
 The DC motors are regular powered motors that can simply be turned on and off, while the servo motors have the functionality of controlling the rotation by a given degree.\\
 The servo motors have a speed of 160-170 RPM, and a rotation precision of 1 degree.
 In comparison, the DC motor has a maximum speed of 380 RPM, making it faster but allowing less control of the rotation.
 
-\subsection{Angle sensor}
+\subsection*{Angle sensor}
 The angle sensor measures axle rotation position and rotation speed.
 It allows measuring both the absolute angle, meaning the rotation of an axle from 0 to 359 degrees with an accuracy of 1 degree.\\
 Likewise, it can measure an accumulated angle, which is the accumulated multiple rotation angle measured since it was last reset.
-Finally the angle sensor can calculate the current RPM with a rotation rate from 1 to 1000 RPM, with an accuracy of 1 RPM.
\ No newline at end of file
+Finally the angle sensor can calculate the current RPM with a rotation rate from 1 to 1000 RPM, with an accuracy of 1 RPM.
+
+\subsection*{Summary of sensors}
+As seen in this section, the NXT platform offers a large variety of sensors that can be useful for the construction of the device.
+The next challenge that must be illuminated is that of managing systems in real time.
+%TODO Maybe we should do some actual testing of the components we use, such as how far it can sense objects and how effeciently 
```

Comment:
Discuss: Maybe we should do some actual testing of the components we use, such as how far it can sense objects and how efficiently

Reply:
Thomas did talk about it a lot. Maybe that was actually a bit of a hint

Reply:
but we don't need to use most of these sensors. we tested the motors we are going to use

Reply:
maybe we should document the testing?